src/test/java/uk/gov/hmcts/reform/sandl/snlevents/actions/hearing/VacateHearingActionTest.java

```diff
@@ -160,6 +152,20 @@ public void act_shouldNotChangeHearingStatuses() {
         assertThat(action.hearing.getStatus().getStatus()).isEqualTo(hearingStatusConfig.getStatus());
     }
 
+    @Test
+    public void act_shouldSetHearingPartSessionIdToNull() {
+        action.getAndValidateEntities();
+        action.act();
+        ArgumentCaptor<Hearing> captor = ArgumentCaptor.forClass(Hearing.class);
+        Mockito.verify(hearingRepository).save(captor.capture());
+        captor.getValue().getHearingParts().forEach(hp -> {
+            if (hp.getStatus().getStatus().equals(Status.Vacated)) {
```

Comment:
in all the other cases shouldn't we assertNotNull?

Reply:
Good point!